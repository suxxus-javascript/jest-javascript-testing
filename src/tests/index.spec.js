jest.mock('tests/__mocks__/isomorphic-fetch');
require('tests/main.js');
