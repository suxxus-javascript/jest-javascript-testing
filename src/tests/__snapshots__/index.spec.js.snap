// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test examples should be 6 1`] = `6`;

exports[`test examples should return a number 1`] = `1`;
